docs/community/licensing.md

```diff
@@ -1,941 +1,10 @@
 # Licensing
 
 napari is distributed under the BSD-3-Clause license, a copy of which is available
-[here][repo_license].
+[here][https://github.com/napari/napari/blob/latest/LICENSE].
 
+```{include} ../../LICENSE
 ```
```

Comment:
For consistency, might be worth doing

````suggestion
```{include} ../../LICENSE
:literal: true
```
````

Reply:
Done! (on both includes) I am curious what's the difference? Or maybe I shouldn't have added the literal on the 2nd one?